src/app/stack/models/stack-report.model.ts

```diff
@@ -127,12 +168,12 @@ export class ComponentConflictUnknownModel {
 }
 
 export class UnknownLicensesModel {
-    really_unknown: Array<ReallyUnknownLicenseModel> = [];
+    unknown: Array<ReallyUnknownLicenseModel> = [];
     component_conflict: Array<ComponentConflictUnknownModel> = [];
 }
 
 export class StackLicenseAnalysisModel {
-    f8a_stack_licenses: Array<string> = [];
+    f8a_stack_licenses: Array<string> = []; // commented 
```

Comment:
f8a_stack_licenses has been renamed to `recommended_licenses`

Reply:
fixed